src/apiutils.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** @module apiutils */
+/**
+ * @fileoverview Contains util functions for api endpoints.
+ */
+
```

Comment:
@siriscac When I try to combine these two jsdoc comments,the documentation generated is not coming up properly.
Can we leave it as such. I promise the  generated doc will be very useful for a new contributor. Please consider..